test/global.test.js

```diff
@@ -14,6 +14,8 @@ test('It should set the default headers', async (t) => {
     reply.send({ hello: 'world' })
   })
 
+  await fastify.ready()
```

Comment:
there is no need for this

Reply:
Removed and cleaned up a test that was not relevant anymore with this implementation change. :smile: